src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (String data: inputData) {
+            if (data.equals("type,fruit,quantity")) {
+                continue;
+            }
+            String[] parts = data.split(",");
+            String operationCode = parts[0].trim();
+            String fruitName = parts[1].trim();
+            int quantity = Integer.parseInt(parts[2].trim());

```

Comment:
You are not handling the case where the quantity might not be a number. What if the data contains a string that cannot be parsed to an integer? This can lead to a NumberFormatException. Consider adding a try-catch block around the Integer.parseInt() call to handle this exception.